geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/DataUtils.groovy

```diff
@@ -34,14 +35,27 @@ IProcess joinTables() {
             inputTableNamesWithId.each { key, value ->
                 if (alias == "a") {
                     columnKey = "$alias.${value}"
-                    columns = datasource.getTable(key).columnNames.collect {
-                        alias + "." + it
```

Comment:
This add-on must be set in a separate PR.

Reply:
OK I will try to make a diff between this commit and 841781e

Reply:
OK I did it manually but it should be OK